### DISCUSSION TITLE
Flow branch based on last node run result

### DISCUSSION THREAD

**Discussion by hialvaro** [windmill-labs/windmill#1729] [Questions]
I would like to be able to branch a flow based on the last node run result; 

if it fails do X, if successful do Y. Is this possible?

**Comment by rubenfiszel** [comment]
I would strongly suggest to have the last node run use: try/catch and then return a different result based on that, then branch based on that result. If not possible, you can simply use the error handler, and make it recover the flow by having it not rethrow an error. In which case, it will continue with the result of the error handler.
